engine/runtime/src/main/scala/org/enso/compiler/phase/StubIrBuilder.scala

```diff
@@ -30,7 +30,7 @@ object StubIrBuilder {
     val consNames = conses.keys.map(_.toLowerCase()).toSet
     val definedConstructors: List[BindingsMap.Cons] =
       conses.toList.map { case (name, cons) =>
-        BindingsMap.Cons(name, cons.getArity)
+        BindingsMap.Cons(name, cons.getArity, allFieldsDefaulted = false)
```

Comment:
any reason to name the parameter here?

Reply:
I think it is more readable.

Reply:
Fair, just didn't in the other usage places.

Reply:
Yeah intellij does it automatically sometimes and I tend to leave it when it does :P